helpers/queries/virtual-protocol.sql

```
@@ -159,41 +179,56 @@ WITH
     -- -- Sol prototype fees
     -- -- Sol sentient (1% of trading volume)
-    -- SELECT 
+    -- SELECT
     --     'solana' as chain,
     --     ( COALESCE(spf.amt, 0) + COALESCE(sv.base_token_amount * 0.01, 0) ) as virtual_fees,
     --     0 as cbbtc_fees
     -- FROM sol_prototype_fees spf
     -- CROSS JOIN sol_volume sv
-)
\ No newline at end of file
+)
```

SQLFluff (4.2.2)

[error] Derived table must have an alias.

(AL10)